org.testeditor.web.backend.persistence/src/main/java/org/testeditor/web/backend/testexecution/TestSuiteResource.xtend

```diff
@@ -134,7 +134,7 @@ class TestSuiteResource {
 	def Response launchNewSuiteWith(List<String> resourcePaths) {
 		val suiteKey = new TestExecutionKey("0") // default suite
 		val executionKey = statusMapper.deriveFreshRunId(suiteKey)
-		val builder = executorProvider.testExecutionBuilder(executionKey, resourcePaths)
+		val builder = executorProvider.testExecutionBuilder(executionKey, resourcePaths, '') // commit id unknown
```

Comment:
you could overload the method to make the last argument optional. Unless you intentionally didn't do this, because not providing a commit id here is only a temporary solution(?)

Reply:
I don't want it to be optional hence no overloading. I will add the commit id to the lanch endpoint in time ...